src/Files.App/Helpers/Application/AppLifecycleHelper.cs

```diff
@@ -109,23 +111,25 @@ public static async Task CheckAppUpdate()
 		}
 
 		/// <summary>
-		/// Configures AppCenter service, such as Analytics and Crash Report.
+		/// Configures Exceptionless service, such as Analytics and Crash Report.
 		/// </summary>
-		public static void ConfigureAppCenter()
+		public static void ConfigureExceptionless()
 		{
 			try
 			{
-				if (!Microsoft.AppCenter.AppCenter.Configured)
+				if (!ExceptionlessClient.Default.Configuration.IsValid)
 				{
-					Microsoft.AppCenter.AppCenter.Start(
-						Constants.AutomatedWorkflowInjectionKeys.AppCenterSecret,
-						typeof(Microsoft.AppCenter.Analytics.Analytics),
-						typeof(Microsoft.AppCenter.Crashes.Crashes));
+					var fakeID = Guid.NewGuid().ToString();
```

Comment:
We might need to store the fake id across sessions but I think that's fine.

Reply:
If this is going to be used we should show it in About to let user know that's session ID.
And output into the debug.log

Reply:
This is actually a user ID, there is another thing we can do which is to generate reference IDs for specific crashes but it's out of scope for this PR.